tests/reference/js_defer_onload.html

```diff
@@ -0,0 +1,20 @@
+<html>
+  <head>
+    <!-- Ensure that onloads in html tags are executed without any errors. -->
+  </head>
+  <body>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/1.10.2/jquery.js"></script>
+    <script type="text/javascript" >
+      window.onload = function () {
+        document.writeln('onload');
```

Comment:
the `document.writeln` in both browser ( chrome and firefox ) will remove all the content of the body, and the result is:

![image](https://user-images.githubusercontent.com/1662759/30581941-3198c22a-9d22-11e7-9eb0-2646913d471f.png)

it's too aggressive for this tests